index.html

```diff
@@ -2,24 +2,87 @@
+					<a href="index.html">Blog</a>
+				</nav>
 			</header>
-			<footer>
-				&copy; Footer Goes Here
+			
+			<div class="clearfix">
+				<section class="main">
+					<h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+					
+					<img src="images/blog_photo1.jpg" alt="rowing photo">
+
+					<p class="maintext"><span>L</span>orem ipsum dolor sit amet, consectetur adipiscing elit. Donec quis libero eget est suscipit rhoncus quis ac nulla. Vestibulum vel sapien eget erat ornare lacinia. Nunc id mi fermentum, blandit metus ac, dictum justo. Donec congue rutrum justo, vitae posuere dui lacinia eu. In pretium blandit erat, eu lobortis nisi consequat vel. Cras et gravida magna. Duis quis dictum ligula. Fusce sagittis, orci quis porta varius, ante lorem finibus diam, in feugiat sapien ligula at tellus. Integer et enim a risus tristique interdum non ac magna. Proin eget dui est. Suspendisse euismod nunc in dui tempus, sit amet consectetur elit imperdiet. Aenean scelerisque porttitor mi, et sollicitudin enim porta et. Aenean quis lacinia turpis.</p>
+
+					<a class="readmore" href="#">Read more &gt;</a>
+
+					<h2 id="secondtitle">I traveled across the globe thanks to Relaxr's Excel automation</h2>
+
+					<img src="images/blog_photo2.jpg" alt="boats photo">
+
+					<p class="maintext">Morbi eu risus nec eros feugiat hendrerit. Nulla lacinia rhoncus velit, sed tincidunt ante hendrerit eget. Vivamus et tellus arcu. Donec tristique diam ex, sed consectetur lectus blandit ac. Curabitur ac ipsum ac erat iaculis semper. Vestibulum porta quam purus. Etiam eu ipsum iaculis ex consequat fermentum ut a massa. Nullam erat lorem, convallis ut sodales non, facilisis a erat. Praesent sed est dapibus, bibendum felis sed, tincidunt enim. Curabitur ultricies lectus quis faucibus feugiat. Vivamus vitae pellentesque arcu. Donec non nunc at diam rutrum volutpat. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Integer sit amet tellus et ligula molestie mattis eu ac enim.</p>
+					
+					<a class="readmore" href="#">Read more &gt;</a>
+				</section>
+			
+				<section class="sidebar">
+					<nav class="categories">
+						<h3>Categories</h3>
+
+						<ul>
+							<li><a href="#">Success stories</a></li>
+							<li><a href="#">Stats</a></li>
+							<li><a href="#">How tos</a></li>
+							<li><a href="#">Best of</a></li>
+							<li><a href="#">Business tips</a></li>
+						</ul>
+					</nav>
+					
+					<aside>
+						<h3>About Relaxr</h3>
+						
+						<p class="about">Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.</p>
+						
+						<a class="learn" href="#">Learn more</a>
+					</aside>
+					
+					<img class="ad" src="images/ad-unit.png" alt="ad unit">
+				</section>
+			</div>	
+
+			<section class="signup">
+				<a class="button" href="#">Sign Up Now!</a>
+			</section>
+			
+			<footer class="clearfix">
```

Comment:
It doesn't look like anything is floated in here so you can likely remove the `clearfix`